src/multiboot-headers.md

```diff
@@ -282,7 +282,26 @@ Again, we’ll use math to let the computer calculate the sum for us. We add up
 the magic number, the mode code, and the header length, and then subtract it
 from a big number. `dd` then puts that value into this spot in our file.
 
-TODO: https://github.com/intermezzOS/book/issues/28
+> **By the way...**
+>
+> You might wonder why we're subtracting these values from a big number. To answer this we can look at what [the multiboot spec](http://nongnu.askapache.com/grub/phcoder/multiboot.pdf) says about the checksum value in the header:
+>
+> > The field `checksum` is a 32-bit unsigned value which, when added to the other magic fields (i.e. `magic`, `architecture` and `header_length`), must have a 32-bit unsigned sum of zero.
```

Comment:
Let's go ahead an assume an appendix on signed and unsigned numbers and link to it here. I volunteer to write it.